packages/cyberstorm/src/components/MenuItem/MenuItem.module.css

```diff
@@ -26,3 +26,11 @@
 .menuItemLabel {
   font-weight: 500;
 }
+
+.menuItemLabel__fontSize__small {
+  font-size: var(--font-size--m);
+}
+
+.menuItemLabel__fontSize__medium {
+  font-size: var(--font-size--l);
+}
```

Comment:
The classnames and variables used are semantically conflicting.

Reply:
Yeah, saw this also. Thought if I should fix this, but decided the fix should be in the globals.css' end. So I'm leaving this as is for now and it will be corrected in the css purge/refactor.